systemtests/suite1.test.ts

```diff
@@ -192,88 +102,116 @@ describe('Suite 1', () => {
 			page.waitForSelector('[value=" Allow "]')
 		]);
 
+		// When
 		await Promise.all([
 			page.click('input#oaapprove'),
-			page.waitForSelector('.auth-finished')
+			page.waitForNavigation()
 		]);
 
 		const contents = await page.content();
-		const regex = new RegExp('<pre>(.*)</pre>');
+		const regex = new RegExp('<pre .*>(.*)</pre>');
 		const matches = regex.exec(contents) as RegExpExecArray;
 
-		// Then
+		// // Then
```

Comment:
This stood out to me, extremely important to fix.

Reply:
Still sorting it :)